share/spack/gitlab/cloud_pipelines/stacks/e4s-cray-rhel/spack.yaml

```diff
@@ -10,7 +10,8 @@ spack:
 
   packages:
     all:
-      require: '%cce'
+      require:
+      - one_of: ["%cce", "@:"]  # cce as a strong preference; not all packages support it
```

Comment:
This is a conflict, `any_of` is a strong preference. I'll add syntactic sugar for those :slightly_smiling_face: